### PR TITLE
Python <3.5.2 build attempt

### PR DESCRIPTION
**Pull request by smarie**
I may be mistaken, but I believe that at least 3.5.1 is actually supported as is

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ python:
   - "3.5"
   - "3.5.3"
   - "3.5.2"
+  - "3.5.1"
```
Comment (ilevkivskyi) [inline .travis.yml]: You need to select an older distro for this build (try `trusty` or `xenial`).

Comment (ilevkivskyi) [reply .travis.yml]: See e.g. https://github.com/python/mypy/blob/master/.travis.yml#L29

**Comment by smarie** [comment]
For some reason, travis does not seem to re-trigger :(

File: .travis.yml (hunk 1 above)
Comment (smarie) [reply .travis.yml]: done.

**Comment by ilevkivskyi** [comment]
This is an invalid Travis config, it can't parse it, see https://travis-ci.org/ilevkivskyi/typing_inspect/requests?utm_medium=notification&utm_source=github_status

**Comment by smarie** [comment]
Indeed ! Sorry for not being able to notice it, and thanks for the tip

**Comment by ilevkivskyi** [comment]
It looks like build fails because of recently added `is_new_type()` test.

**Comment by smarie** [comment]
3.5.1 is ok.

I am checking 3.5.0 now

**Comment by smarie** [comment]
Well it seems all ok. Shall we add other 3.5.x versions explicitly or is typing module stable starting in 3.5.4 ?

**Comment by ilevkivskyi** [comment]
> Well it seems all ok. Shall we add other 3.5.x versions explicitly or is typing module stable starting in 3.5.4 ?

I think it is relatively stable at that point.

**Pull request merged**